redash/query_runner/pg.py

```diff
@@ -204,20 +204,6 @@ def _get_definitions(self, schema, query):
         build_schema(results, schema)
 
     def _get_tables(self, schema):
-        """
-        relkind constants per https://www.postgresql.org/docs/10/static/catalog-pg-class.html
-        r = regular table
-        v = view
-        m = materialized view
-        f = foreign table
-        p = partitioned table (new in 10)
-        ---
-        i = index
-        S = sequence
-        t = TOAST table
-        c = composite type
-        """
-
```

Comment:
@eradman 

In my opinion, I just personally feel the comment what 'm' means is helpful. But it is just a feeling. I can approve it.

Reply:
fair enough, I'll put part of that comment back